packages/flutter/lib/src/gestures/semantics.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'drag_details.dart';
+
+/// Called when the user taps with a semantics device.
+typedef SemanticsTapCallback = void Function();
```

Comment:
So, will something that is declared as a `GestureTapCallback` be passed to these functions that take a `SemanticsTapCallback`?  If so, then I worry a bit that if we modify GestureTapCallback, then we'd have to know to also change this signature.

Probably not a big deal, since the compiler will complain a lot if that happens, but you might want to put some non-doc comments here about keeping them in sync.

If this is just renaming the type of arguments here (as it looks like it might be), then this is fine.

Reply:
The idea is that information passed by a semantic event might be different from that passed by a pointer event, therefore it makes sense for them to diverge. Actually I was about to change this callback to include a `SemanticsTapDetails`, but didn't because that will be a breaking change for `RenderSemanticsGestureHandler`. I think when it happens (which will be a breaking change anyway) we should _not_ keep them in sync.